test/vesrion_test.cc

```diff
@@ -0,0 +1,12 @@
+#include <gtest/gtest.h>
```

Comment:
SHA256 signature of submodule ensures that gRPC version is ``v1.62.0``, so this test is unnecessary.